lua/astrocommunity/git/blame-nvim/init.lua

```diff
@@ -1,6 +1,7 @@
 return {
   "FabijanZulj/blame.nvim",
-  event = "User AstroGitFile",
+  cmd = "BlameToggle",
+  config = function(_, opts) require("blame").setup(opts) end,
```

Comment:
```suggestion
  opts = {}
```

No need to redefine the default `config` function

Reply:
Thanks, teached me something new :)!